substrafl/strategies/fed_pca.py

```diff
@@ -0,0 +1,107 @@
+from typing import List
+from typing import Optional
+
+import numpy as np
+from numpy import linalg
+
+from substrafl.algorithms.algo import Algo
+from substrafl.exceptions import EmptySharedStatesError
+from substrafl.nodes.aggregation_node import AggregationNode
+from substrafl.nodes.references.local_state import LocalStateRef
+from substrafl.nodes.references.shared_state import SharedStateRef
+from substrafl.nodes.train_data_node import TrainDataNode
+from substrafl.remote import remote
+from substrafl.schemas import FedAvgAveragedState
+from substrafl.schemas import FedAvgSharedState
+from substrafl.schemas import StrategyName
+from substrafl.strategies import FedAvg
+
+
+class FedPCA(FedAvg):
+    """Federated Principal Component Analysis strategy."""
+
+    def __init__(self):
+        super(FedPCA, self).__init__()
+
+        # current local and share states references of the client
+        self._local_states: Optional[List[LocalStateRef]] = None
+        self._shared_states: Optional[List[SharedStateRef]] = None
```

Comment:
```suggestion
    def __init__(self):
        super(FedPCA, self).__init__()
```
The init is exactly the same as FedAvg. I would go for letting it to make it clear, but you don't need to re-set `local_states` and `shared_states` to `None`